src/Microsoft.DotNet.VersionTools/Dependencies/DependencyUpdateTask.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.VersionTools.Automation;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.VersionTools.Dependencies
+{
+    /// <summary>
+    /// A task that can be performed to update dependency versions to values in BuildInfos.
+    /// 
+    /// Like System.Threading.Task, a call to Result blocks on the update being performed.
```

Comment:
I need `UsedBuildInfos` and `ReadableDescriptionLines` before evaluating, but maybe subclassing would work. I'll see how that would go.